src/core_atmosphere/physics/mpas_atmphys_driver_lsm.F

```diff
@@ -16,6 +16,7 @@ module mpas_atmphys_driver_lsm
  use mpas_atmphys_landuse, only: isurban
  use mpas_atmphys_lsm_noahinit
  use mpas_atmphys_lsm_rucinit
+ use mpas_atmphys_lsm_rucinit
```

Comment:
Is wine making me see double lines?

Reply:
@jderrico-noaa We need to have line 18, but no repeat is needed.